lumicks/pylake/detail/imaging_mixins.py

```diff
@@ -72,3 +77,30 @@ def plot(num):
         )
         line_ani.save(file_name, writer=writer)
         plt.close(fig)
+
+
+class FrameIndex:
+    def to_frame_index(self, time, use_start=False) -> Union[int, None]:
```

Comment:
should be private API?

also perhaps `_time_to_frame_index` might be more descriptive?

Reply:
Yes, should definitely be private API. Good catch!